android/app/build.gradle

```diff
@@ -78,7 +78,7 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    enableHermes: false,  // clean and rebuild if changing
+  enableHermes: false,  // clean and rebuild if changing
```

Comment:
Why is there so many Code Style diff? Is there a tool fixing this automatically? If so, could it be run once and for all in a dedicated PR and then ran each time on pre-commit via `lint-staged`?

Reply:
Sure! https://github.com/dis-moi/mobile_poc/issues/29